tests/unit/neptune/new/test_stringify_unsupported.py

```diff
@@ -411,6 +412,7 @@ def test_assign__empty_list__reassign(self, run):
 
         assert run["regular"].fetch() == run["stringified"].fetch()
 
+    @freeze_time("2024-01-01T12:34:56.123456Z")
```

Comment:
just to clarify. `freeze_time` is necessary because of change in `src/neptune/internal/backends/neptune_backend_mock.py`?

Reply:
Yes